docbot/action.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Documentation Bot'
+description: 'Automatically label pull requests based on the checked task list'
+author: 'Max Xu'
```

Comment:
how about adding your GitHub ID so that others can find you easily if they have any questions?

Reply:
Make sense, the author's email is added.